src/bpftrace.cpp

```diff
@@ -1282,6 +1286,24 @@ uint64_t BPFtrace::resolve_kname(const char *name)
   return addr;
 }
 
+uint64_t BPFtrace::resolve_cgroupid(const char *path)
+{
+  uint64_t cgid;
+  int err, mount_id;
+  file_handle *handle = reinterpret_cast<struct file_handle *>(
```

Comment:
The `struct` keyword inside `reinterpret cast` is unnecessary.

Reply:
ok, I'll remove it.